test/Map_object.html

```diff
@@ -4,14 +4,14 @@
 <script src="../js/mapModel.js"></script>
 <script>
 function test(){
-let tile = new Tile("Water", 0,0, 0, "Item");
+let tile = new Tile("Meadow", 0,0, 0, "Item");
```

Comment:
I think this should be Meadow without quotes. I'd still approve the pull request and we'll make tiny fixes like this later.